bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -36,7 +36,7 @@ trait Asterism {
     * only possibilities.
     * TODO: should we Church encode this instead?
     */
-  def targets: Target \/ (SiderealTarget, SiderealTarget)
+  def targets: Target \/ (Target, Target)
```

Comment:
This defeats the purpose of the `target` method … things like `.isSidereal` depend on knowing that if there's more than one target they have to be sidereal.